### DISCUSSION TITLE
Demo Stream For Testing

### DISCUSSION THREAD

**Discussion by mitchross** [blakeblackshear/frigate#1740] [Ask A Question] [answered]
Is there a public stream that can be used for testing frigate if someone doesn't have a rtsp camera?

**Comment by mitchross** [comment, marked as answer]
Solved. Use http://www.insecam.org/

**Comment by R5fan** [comment]
How did you feed that video in to frigate? I have been trying to feed it images for testing, but I have not been successful.

**Reply by mitchross** [reply]
goto a cam, right click inspect. Copy the .mjpeg stream url.
